morpheushelper/cogs/run_code.py

```diff
@@ -0,0 +1,69 @@
+import re
+
+from PyDrocsid.translations import translations
+from PyDrocsid.util import send_long_embed
+from discord import Embed
+from discord.ext import commands
+from discord.ext.commands import Cog, Bot, CommandError, UserInputError
+from requests import RequestException
+
+from colours import Colours
+from emkc_api import Emkc, EmkcAPIException
+
+N_CHARS = 1000
+
+LANGUAGES = (
+    "awk,bash,brainfuck,c,cpp,crystal,csharp,d,dash,deno,elixir,emacs,go,haskell,java,jelly,julia,"
+    "kotlin,lisp,lua,nasm,nasm64,nim,node,osabie,paradoc,perl,php,python2,python3,ruby,rust,swift,"
+    "typescript,zig"
+).split(",")
```

Comment:
Pls make a list out of this

Reply:
pls do it fast

Reply:
> pls do it fast

thank you so much for this constructive comment